common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -13,6 +13,12 @@
 import ddt
 from contracts import contract
 from nose.plugins.attrib import attr
+# For the cache tests to work, we need to be using the Django default
+# settings (not our usual cms or lms test settings) and they need to
+# be configured before importing from django.core.cache
+from django.conf import settings
+if not settings.configured:
```

Comment:
If settings *are* configured here and are cms/lms test settings these tests will fail?

Reply:
Pretty much; that happened during my first attempt at fixing this.  Paver is specifically configured to run the common/lib unit tests (which mostly don't depend on Django) separately from the cms/lms tests.  This test should probably be refactored so as to either not care about which settings are loaded or to be run along with other tests that depend on the same settings being in place, but I'm trying to keep changes minimal for the initial pytest switchover.